src/routes/utility.router.ts

```diff
@@ -0,0 +1,15 @@
+import Router from "koa-router";
+import MetricsController from "../controllers/metrics.controller";
+import ProfileController from "../controllers/profile.controller";
+import UtilityController from "../controllers/utility.controller";
+
+const router = new Router();
+
+router.post("/heapdump", ProfileController.profile);
+router.get("/metrics", MetricsController.exportMetric);
+router.get("/health", UtilityController.healthStats);
+router.get("/transformerBuildVersion", UtilityController.buildVersion);
```

Comment:
Let's change it to just `buildVersion`. If we have to be backward compatible, keep both but mark the current one as deprecated.

Reply:
Internally service only right, we can do it for now, why maintain both

Reply:
I'm not sure which all places this is used. Hence, was suggesting to support both for backward compatibility.

Reply:
Refactored